Makefile

```diff
@@ -13,3 +13,25 @@ install:
 
 test:
 	python -m poetry run python -m pytest -v tests
+
+build:
+	docker build -t ppp-project-2 .
+
+run:
+	@if [[ $$(docker ps -aq) ]]; then \
+        echo "Containers Already Running";\
+		docker ps;\
+    else \
+        echo "Booting Container"; \
+		docker run -d -p 5000:5000 --name ppp-project-2 ppp-project-2;\
+		sleep 5;\
+		open http://localhost:5000;\
```

Comment:
Nice little script here to open the app in a browser tab. Worth discussing, whether this is desirable for folks.

Reply:
will float in the channel